konfera/templates/konfera/event_speakers.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'konfera/base.html' %}
+
+{% block title %}<h1>{{ event }} Speakers</h1>{% endblock %}
```

Comment:
@TomasHo @Redpoint1 is right, I thing we can have `<title>` before `{% block %}` in base template and in view templates we just insert text into `{% block title %}` section. This way we avoid html in this block.